apps/portal-web/src/pageComponents/dashboard/CustomizableLogoAndText.tsx

```diff
@@ -0,0 +1,41 @@
+import { Divider } from "antd";
+import Image from "next/image";
+import React from "react";
+import { publicConfig } from "src/utils/config";
+
+
+interface Props {
+  hostname: string | undefined;
+}
+export const CustomizableLogoAndText: React.FC<Props> = ({ hostname }) => {
+
+  return (
+    <div>
+      <div style={{ position: "relative", width: "100%", paddingBottom: "8%" }} >
```

Comment:
we are mostly using [styled-components](https://styled-components.com/) for styling.